data/se.sjoerd.Graphs.appdata.xml.in.in

```diff
@@ -51,10 +51,11 @@
     </screenshot>
   </screenshots>
   <releases>
-    <release version="1.7.0" date="2023-12-14">
+    <release version="1.7.0" date="2024-01-10">
       <description translatable="no">
         <p>Version 1.7 marks our biggest release yet. With a migration to GNOME 45 and a new UI overhaul with a revamped style editor.
-           Other major highlights include a new curve fitting functionality, the ability to open projects directly, and support for gestures in the canvas.
+           Other major highlights include a new curve fitting functionality, the ability to open projects directly, and support for gestures in the canvas. A full changelog
```

Comment:
One other small nitpick I found then its good from my side
```suggestion
           Other major highlights include a new curve fitting functionality, the ability to open projects directly and support for gestures in the canvas. A full changelog
```

Reply:
It's an Oxford comma, which is not technically wrong (nor correct). But in this case I think we can do without, it's only really needed if there's any confusion otherwise.

![image](https://github.com/Sjoerd1993/Graphs/assets/68477016/299c7bfe-861c-4aa3-8a90-055a9c35337c)